src/components/DonationWidget/components/DonationWizard.tsx

```diff
@@ -0,0 +1,43 @@
+import { ButtonHTMLAttributes, DetailedHTMLProps } from 'react';
+import tw from 'twin.macro';
+import styled from 'styled-components';
```

Comment:
Mmmm I wasn't aware we want to use styled-components. What about sticking to what we are doing in the homepage. Also, this might affect our bundle size too

Reply:
I do understand your concern, nevertheless, I see some benefits I'd like to expose first so maybe you and @arathjz can give me feedback around that. The things I wanted to improve when choosing `styled-components` were:

1. **Improve maintainability of the code by having a central place where to update component styles**:
So for instance, if I need a `Title` I can do [something like this](https://github.com/debtcollective/home/blob/d98518de720a769a3216eed530e68e854326d196/src/components/DonationWidget/components/DonationWizard.tsx#L41) and [apply like this](https://github.com/debtcollective/home/blob/d98518de720a769a3216eed530e68e854326d196/src/components/DonationWidget/components/DonationPaymentForm.tsx#L19) instead having a need of copy and paste a `div` with several classes, when we need to update the dev need to update on every file and make sure all classes are present, this later approach is prone to errors as the dev may apply changes and forget to update one of the usages _I see this very easily happening_

2. **Improve the handling of the logic needed in order to affect component styling**:
As the widget is mean to be isolated check out the difference of dealing with [Button with styled-components](https://github.com/debtcollective/home/blob/d98518de720a769a3216eed530e68e854326d196/src/components/DonationWidget/components/DonationWizard.tsx#L26) and [without styled-component](https://github.com/debtcollective/home/blob/development/src/components/Button/index.tsx#L35). Whenever we have components that rely on JavaScript for their style, Styled Components gives control of those states back to CSS instead of using a multitude of conditional class names.

With all of that being said, we can achieve both without `styled-components` by: Having the same file but exporting pure react components with `children` within and when a need to rely on JS for styles we can encapsulate the logic within those components, while the option I just described is perfectly valid I found it much more verbose and somewhat awkward.

Reply:
I will drop this comment on our slack channel and we can sync over this decision. If we decide to drop the library despite the above mentioned I can introduce a new PR to remove `styled-components` and adopt the alternative I have exposed _(unless there are other alternatives you guys have in mind.)_

Reply:
I think we can do all of this with [css modules](https://www.gatsbyjs.com/docs/css-modules/), adding a dependency that adds 13kb (gziped) to the bundle size is not worth it for just one component, remember we want to use this component in different contexts not only on this page (ex. as a third party library on other sites).

I think you should have been vocal about adding this dependecy before doing the work, I'm not sure if with the time we have now is a good idea to remove it at this point.